src/tbots.py

```diff
@@ -0,0 +1,92 @@
+#!/opt/tbotspython/bin/python3.8
+
+import os
+import sys
+import iterfzf
+from subprocess import PIPE, run, check_call
+import argparse
+from thefuzz import fuzz
+from thefuzz import process
+
+# thefuzz is a fuzzy string matcher in python
+# https://github.com/seatgeek/thefuzz
+#
+# It returns a match ratio between the input and the choices
+# This is an experimentally determined threshold that works
+# for our bazel commands
+THEFUZZ_MATCH_RATIO_THRESHOLD = 50
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description="Run stuff")
+
+    parser.add_argument("action", choices=["build", "run", "test"])
+    parser.add_argument("search_query")
+    parser.add_argument("-v", "--enable_visualizer", action="store_true")
+    parser.add_argument("-s", "--stop_ai_on_start", action="store_true")
+    parser.add_argument("-p", "--print_command", action="store_true")
```

Comment:
why not always print out the command that it matched to for clarity? Also, I tried `./tbots.py test corner -s -v -p` and it matched to `bazel test //software/gui:geometry_conversion_test --cache_test_results=false --test_arg="--stop_ai_on_start" --test_arg="--enable_visualizer"
`

Reply:
hmm not sure what you mean? it does print the matched command rn for both selection and thefuzz

re: mismatch, yeah idk what the solution there is other than we might need to put a bit more effort in the query tbh, open to suggestions

Reply:
Perhaps if this is happening often we would need to increase `THEFUZZ_MATCH_RATIO_THRESHOLD` to narrow down the search result accuracy

Reply:
ok, this is fine